README.md

```diff
@@ -45,27 +34,17 @@ gsheets --key=1iOqNjB-mI15ZLly_9lqn1hCa6MinqPc_71RoKVyCFZs --title=foobar --pret
 
 ### Compatibility Note
 
-gsheets uses the [Fetch API](https://developer.mozilla.org/en/docs/Web/API/Fetch_API) and [Promises](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Promise). Depending on your environment, you'll need to polyfill those. Recommendations:
+gsheets uses the [Fetch API](https://developer.mozilla.org/en/docs/Web/API/Fetch_API). Depending on your environment, you'll need to polyfill this. Recommendations:
 
 - [isomorphic-fetch](https://github.com/matthew-andrews/isomorphic-fetch)
-- [es6-promise](https://github.com/stefanpenner/es6-promise)
-
-For direct usage in the browser, there's a pre-built version of gsheets which the polyfills at [gsheets.polyfill.js](https://unpkg.com/gsheets@next/gsheets.polyfill.js) and [gsheets.polyfill.min.js](https://unpkg.com/gsheets@next/gsheets.polyfill.min.js).
 
 ## Features
 
 * Plain JS/JSON data. No 'models'. Just use `.map`, `.filter` etc.
 * Correct handling of numeric cells (no formatted strings for numbers!)
 * Empty cells are converted to `null`
-* A bit of metadata (i.e. when a spreadsheet was updated)
-* Empty rows are omitted
-* Correct handling of empty worksheets
-
-## Non-features
-
-* Authorization (only works with [published spreadsheets](https://support.google.com/docs/answer/37579?hl=en&ref_topic=2818999))
-* Querying, ordering, updating
-* Caching. Use a reverse proxy or implement your own caching strategy. I recommend this strongly since Google's API isn't the fastest and you don't want to hit rate limits.
```

Comment:
especially `Authorization` being a non feature is still true. IMO leave this in there – sheets still need to be published.